bin/GCStress/WeightedTable.h

```diff
@@ -13,14 +13,14 @@ template <class T>
 class WeightedTable
 {
 public:
-    WeightedTable() :
+    WeightedTable() noexcept :
```

Comment:
>noexcept [](start = 20, length = 8)

Did you mean to add this?

Reply:
Yeah, it was that or a longer statement to disable the warning for just that line. This took fewer keypresses, and won't complicate any future efforts to go over all the warnings we've disabled.